tools/packaging/release/generate_vendor.sh

```diff
@@ -33,7 +33,7 @@ create_vendor_tarball() {
 	# This is very much error prone in case we re-structure our
 	# repos again, but it's also used in a few other places :-/
 	pushd ${repo_dir}
-		for i in `find . | grep Cargo.toml`; do
+		for i in `find . | grep Cargo.lock`; do
```

Comment:
Why is this not

```
for in in $(find . -name 'Cargo.lock'); do
```

??? Why use two processes when one would do.

Also, can you update the comment above to explain what you are looking for. My understanding is that you only have the `Cargo.lock` for top-level components (as opposed to downloaded crates). Is that the intent?

Reply:
I don't understand enough to give you an answer.
I actually asked for something quite similar a few comments above, see https://github.com/kata-containers/kata-containers/pull/3974#issuecomment-1081175702

Reply:
Oh, the comment shouldn't be there, the comment actually applies to the "repo_dir=" var assignment.